tests/Unit/NumericalAlgorithms/Spectral/Test_Spectral.cpp

```diff
@@ -242,7 +246,7 @@ template <Spectral::Basis BasisType, Spectral::Quadrature QuadratureType,
 void test_exact_extrapolation_impl(const Function& max_poly_deg) {
   const DataVector target_points{-1.66, 1., 1.5, 1.98, 2.};
   // Errors are larger when extrapolating outside of the original grid:
-  const double eps = 1.e-9;
+  const double eps = 1.e-5;
```

Comment:
Oof, this is getting pretty darn large. Do you know for what number of grid points? We could keep the value(s) we test for  the same as before, which might help.

Reply:
Starting from the original `eps` being `1.e-9`, the number of points can be raised to 13 and this test will still pass.
14 pts requires `1.e-8`,
15 pts requires `1.e-8`,
16 pts requires `1.e-7`,
17 pts requires `1.e-7`,
18 pts requires `1.e-6`,
19 pts requires `1.e-6`,
20 pts requires `1.e-5` - so I will be restricted the test values to `12`, but let me know if I should do anything else!

Reply:
I think this would be very useful to have in the test as a comment, eg
```cpp
 // 14 pts requires 1.e-8
 // 15 pts requires 1.e-8
 // 16 pts requires 1.e-7
 // 17 pts requires 1.e-7
 // 18 pts requires 1.e-6
 // 19 pts requires 1.e-6
 // 20 pts requires 1.e-5
```
Do you mind adding that quickly? That'll hopefully save someone in the future time trying to figure out what's going on if they increase the number of points